### PR TITLE
Create National Cycle Network layer

### PR DESCRIPTION
**Pull request by Pete-Y-CS**

### CODE REVIEW

**Comment by dabreegster** [comment]
This isn't in LayerControls, so it's not accessible yet

File: src/lib/browse/layers/lines/NationalCycleNetwork.svelte [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  National Cycle Network
+  <span slot="right">
+    <HelpButton>
+      <p>
+        Data from Sustrans about the <ExternalLink
+          href="https://data-sustrans-uk.opendata.arcgis.com/"
+        >
+          National Cycle Network
+        </ExternalLink>, downloaded on 22/01/2024.
+      </p>
```
[the remark this answers is not in the document]
Comment (Pete-Y-CS) [reply src/lib/browse/layers/lines/NationalCycleNetwork.svelte:35]: Not sure what to put for this, will look into it today unless you know the answer readily

[the remark this answers is not in the document]
Comment (dabreegster) [reply src/lib/browse/layers/lines/NationalCycleNetwork.svelte:35]: Hidden in the arcgis page. We can use `<OsOglLicense />`
Screenshot from 2024-01-23 10-25-32

**Review by dabreegster — APPROVED** [review]
Nice!

**Pull request merged**